content/en/docs/guides/custom-load-balancing.md

```diff
@@ -42,6 +44,16 @@ call path
 The exact steps vary by language, see the language support section for some
 concrete examples in your language.
 
+```mermaid
+flowchart TD
+NR(Name Resolver) -->|Provides addresses &\nLB config| LB(Load Balancer)
+LB --> |Provides a picker| C(Channel)
+C -->|Picks\na subchannel| P(Picker)
```

Comment:
Would it be better to say the channel "requests a subchannel" from the picker?

Reply:
Yes!